Formula/dovecot.rb

```diff
@@ -3,6 +3,7 @@ class Dovecot < Formula
   homepage "https://dovecot.org/"
   url "https://dovecot.org/releases/2.3/dovecot-2.3.10.1.tar.gz"
   sha256 "6642e62f23b1b23cfac235007ca6e21cb67460cca834689fad450724456eb10c"
+  license "LGPL-2.1-or-later"
```

Comment:
```suggestion
  license all_of: ["BSD-3-Clause', "LGPL-2.1-or-later", "MIT", "Unicode-DFS-2015", :public_domain]
```
Blocked by PR https://github.com/Homebrew/brew/pull/8260